### ISSUE TITLE
GraphQL error when providing a default value for variables

### ISSUE THREAD

**Issue by MichaelDeBoey** [Youshido/GraphQL#148]
Like @fritz-gerneth already mentioned in [#144 (comment)](https://github.com/Youshido/GraphQL/issues/144#issuecomment-302694031), you get an error when providing a default value for a variable

```js
query ($username: String = "GovSchwarzenegger"){
  reddit {
    user(username: $username) {
      username
      commentKarma
      createdISO
    }
  }
}
```

```js
{"errors": [
    {
      "message": "Can\\t recognize token type",
      "location": [
        {
          "line": 1,
          "column": 25
        }
      ]
    }
]}
```

CC: @viniychuk

**Comment by fritz-gerneth** [comment]
Would be great if you could check #150 if this works for you or if I missed some parts :)

**Comment by MichaelDeBoey** [comment]
@fritz-gerneth I'm not the one who's responsible for the server so I can't test it out unfortunately 😕

**Comment by viniychuk** [comment]
@fritz-gerneth Thanks! I just added additional test for parser, merging.

**Comment by viniychuk** [comment]
@fritz-gerneth Thanks! I just added additional test for parser, merging.

**Comment by viniychuk** [comment]
fixed in 1.4.3.4
